airbyte-cdk/python/unit_tests/sources/declarative/stream_slicers/test_datetime_stream_slicer.py

```diff
@@ -358,7 +360,7 @@ def test_stream_slices(
     )
     stream_slices = slicer.stream_slices(SyncMode.incremental, stream_state)
 
-    assert expected_slices == stream_slices
+    assert stream_slices == expected_slices
```

Comment:
Every test does `assert expected == actual` but PyTest assertions should be `assert actual == expected`. I have only changed this one but it could be interesting to write new tests using this convention